firedrake/cython/dmcommon.pyx

```diff
@@ -1154,31 +1386,27 @@ def validate_mesh(PETSc.DM plex):
 
     from mpi4py import MPI
 
-    pStart, pEnd = plex.getChart()
-    cStart, cEnd = plex.getHeightStratum(0)
+    get_chart(dm.dm, &pStart, &pEnd)
+    get_height_stratum(dm.dm, 0, &cStart, &cEnd)
 
     CHKERR(PetscBTCreate(pEnd - pStart, &seen))
     nseen = 0
     # Walk the cells, counting the number of points we can traverse in
     # the closure.
     for c in range(cStart, cEnd):
-        CHKERR(DMPlexGetTransitiveClosure(plex.dm, c,
-                                          PETSC_TRUE,
-                                          &nclosure,
-                                          &closure))
+        get_transitive_closure(dm.dm, c, PETSC_TRUE, &nclosure, &closure)
         for ci in range(nclosure):
             p = closure[2*ci]
             if not PetscBTLookup(seen, p):
                 nseen += 1
                 PetscBTSet(seen, p)
 
     if closure != NULL:
-        CHKERR(DMPlexRestoreTransitiveClosure(plex.dm, 0, PETSC_TRUE,
-                                              NULL, &closure))
+        restore_transitive_closure(dm.dm, 0, PETSC_TRUE, &nclosure, &closure)
     CHKERR(PetscBTDestroy(&seen))
 
     # Check validity on all processes
-    valid = plex.comm.tompi4py().allreduce(nseen == pEnd - pStart,
+    valid = dm.comm.tompi4py().allreduce(nseen == pEnd - pStart,
                                            op=MPI.LAND)
```

Comment:
Indent wrong, rewrap onto previous line.

Reply:
thanks, changed. Surprised this didn't get caught by `make lint`